misc/formatter/modules/formatter-core/src/test/resources/declarations/class-definition/source/class_definition_declaration_15.bal

```diff
@@ -1,32 +1,32 @@
       client
    readonly
          distinct
-
-     class  Foo
+            class  Foo
```

Comment:
same for this entire file

Reply:
This is a source file, so should it be changed as well?

Reply:
ack